FINS/FINS-IOC-01App/Db/sans2d-vac-status.template

```diff
@@ -7,28 +7,53 @@
 
 record(mbbi, "$(FULL_PREFIX)$(SYSTEM):STATUS") {
     field(DESC, "$(SYSTEM) Status")
-    field(DTYP, "asynInt32")
-    field(INP,  "@asyn(PLC, $(ADDR), 5.0) FINS_DM_READ")
-    field(SCAN, "1 second")
-    field(NOBT, 4)
+    field(INP, "$(FULL_PREFIX)$(SYSTEM):STATUS:_SET_UNKNOWN CP MS")
     
+    field(ZRVL, "0")
     field(ONVL, "1")
     field(TWVL, "2")
     field(FRVL, "4")
     field(EIVL, "8")
     
+    field(ZRST, "ERROR: STATUS UNKNOWN")
     field(ONST, "ATMOSPHERE")
     field(TWST, "VAC DOWN")
     field(FRST, "AT VACUUM")
     field(EIST, "VENTING")
-    
-    field(SIML, "$(P)$(Q)SIM")
-    field(SIOL, "$(P)$(Q)SIM:$(SYSTEM):STATUS")
+
+    field(ZRSV, "MINOR")
 
     info(archive, "VAL")
     info(INTEREST, "HIGH")
 }
 
+record(longin, "$(FULL_PREFIX)$(SYSTEM):STATUS:_RAW") {
+    field(DESC, "Raw $(SYSTEM) status from device")
+    field(SCAN, "1 second")
+    field(DTYP, "asynInt32")
+    field(INP,  "@asyn(PLC, $(ADDR), 5.0) FINS_DM_READ")
+}
+
+record(mbbi, "$(FULL_PREFIX)$(SYSTEM):STATUS:_SET") {
+    field(DESC, "$(SYSTEM) Status")
+    field(INP, "$(FULL_PREFIX)$(SYSTEM):STATUS:_RAW CP MS")
+    field(NOBT, 4)
+    
+    field(ONVL, "1")
+    field(TWVL, "2")
+    field(FRVL, "4")
+    field(EIVL, "8")
+
+    field(SIML, "$(P)$(Q)SIM")
```

Comment:
I think the recsim PV should be attached to the PV reading from hardware, as we should add recsim at the earliest layer possible so recsim behaves as similar as possible to devsim.

Reply:
Though I agree that would be ideal, as said in another comment there is an issue when simulating asyn records with this IOC. I believe it may be because we never get a return from the asyn call and as such the setting of the simulation value is blocked.